styles/signals.mapcss

```diff
@@ -69,10 +69,18 @@ way["railway:lzb"=yes].tracks
 	z-index: 3;
 	color: red;
 }
+way["railway:atb"=yes].tracks
+way["railway:atb-eg"=yes].tracks
+way["railway:atb-ng"=yes].tracks
+way["railway:atb-vv"=yes].tracks
+{
+	z-index: 4;
```

Comment:
This should still be lowered to 2 or 3.

Reply:
In my opinion 4 is okay so that every rule has a defined priority. If you use 2 or 3 here, the style does not define the rendering order, so it may happen that e.g. at the borders the lines are rendered randomly.